vnext/Shared/InspectorPackagerConnection.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <WinRTWebSocketResource.h>
+#include <jsinspector/InspectorInterfaces.h>
+
+namespace Microsoft::ReactNative {
+
+class InspectorPackagerConnection : public std::enable_shared_from_this<InspectorPackagerConnection> {
```

Comment:
>InspectorPackagerConnection [](http://example.com/codeflow?start=6&length=27)

Can we put 'noexcept' to any of these methods? The 'noexcept' method size is ~10 lower than without 'noexcept'. It affects the binary size.

Reply:
I've made these API noexcept. AFAIK none of the code in it should throw. Websocket failures should callback the error handlers.